pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/EACInformation/__specs__/EACInformation.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import '@testing-library/jest-dom'
+
+import { render, screen } from '@testing-library/react'
+
+import EACInformation from '../EACInformation'
+import type { History } from 'history'
+import React from 'react'
+import { Router } from 'react-router-dom'
+import { createBrowserHistory } from 'history'
+
+describe('EACInformation', () => {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Est ce qu'on test les feature toggles dans le front ? 
Si oui tu peux mettre un test avec le FF à `False`

Reply:
Yes je teste le FF dans VenueCreation.spec.tsx directement en testant le cas on et off :)